brightlayer-ui/angular-showcase-demo
# blui 2876 add PR labeler for master PRs

Fixes # .


#### Changes proposed in this Pull Request:
- add PR labeler configs that should place our "master" label on pull request merging to master branch.
- a dev or hotfix pull request to master triggers the labeler to place "master label" on the PR
- second part of job has a "run if" the github.head_ref is master and to echo a message "Are you sure this PR is for master branch"
- ```lastly``` I have no clue if this config will work so its trial by error


#### Screenshots:


#### To Test:
- after merge to dev, create a hotfix branch with anything and open PR against master branch.
- verify master label is placed on PR
- verify a message appears
- unsure if the configs will update the repo actions and this might need to move in to master as well and then we can test
